includes/mail/EmailNotification.php

```diff
@@ -294,7 +294,9 @@ public function actuallyNotifyOnPageChange(
 				continue;
 			}
 			$user = User::newFromName( $name );
-			$this->compose( $user, self::ALL_CHANGES, $messageCache );
+			if !( $user instanceof Boolean) {
```

Comment:
```suggestion
			if ( !is_bool( $user ) ) {
```
if we must check for Boolean, but I think this should be checking for user, no?
```suggestion
			if ( $user instanceof User ) {
```